src/main/java/org/nrnb/gsoc/enrichment/tasks/FilterEnrichmentTableTask.java

```diff
@@ -1,113 +1,121 @@
 import org.cytoscape.work.ObservableTask;
 import org.cytoscape.work.ProvidesTitle;
 import org.cytoscape.work.TaskMonitor;
 import org.cytoscape.work.Tunable;
 import org.cytoscape.work.json.JSONResult;
-import org.cytoscape.work.util.BoundedDouble;
 import org.cytoscape.work.util.ListMultipleSelection;
-import org.cytoscape.service.util.CyServiceRegistrar;
-
-
+import org.nrnb.gsoc.enrichment.model.EnrichmentTerm;
 import org.nrnb.gsoc.enrichment.model.EnrichmentTerm.TermSource;
 import org.nrnb.gsoc.enrichment.ui.EnrichmentCytoPanel;
 import org.nrnb.gsoc.enrichment.ui.EnrichmentTableModel;
+import org.nrnb.gsoc.enrichment.utils.ModelUtils;
 
-
-public class FilterEnrichmentTableTask extends AbstractTask implements ObservableTask {
-
-  private CyApplicationManager applicationManager;
-	private EnrichmentCytoPanel enrichmentPanel;
-	private CyNetwork network;
-	private CyTable filteredEnrichmentTable;
-  private CyServiceRegistrar registrar;
-  private List<TermSource> categoryFilter = TermSource.getValues();
-	private final Logger logger = Logger.getLogger(CyUserLog.NAME);
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.TreeSet;
 
 
-  @Tunable(description = "Select categories",
-	         tooltip = "Select the enrichment categories to show in the table",
-	         longDescription = "Select the enrichment categories to show in the table. Use \"All\" to remove the filtering.",
-	         exampleStringValue = "GO Process",
-	         gravity = 1.0)
-	public ListMultipleSelection<TermSource> categories = new ListMultipleSelection<>(TermSource.getValues());
+public class FilterEnrichmentTableTask extends AbstractTask implements ObservableTask {
 
-	//public boolean removeOverlapping = false;
+    @Tunable(description = "Select categories",
+            tooltip = "Select the enrichment categories to show in the table",
+            longDescription = "Select the enrichment categories to show in the table. Use \"All\" to remove the filtering.",
+            exampleStringValue = "GO Process",
+            gravity = 1.0)
+    public ListMultipleSelection<TermSource> categories = new ListMultipleSelection<>(TermSource.getValues());
+
+    @Tunable(description = "Select evidence code",
+            tooltip = "Select the evidence codes to show",
+            exampleStringValue = "GO:0005737",
+            gravity = 1.0
+    )
+    public ListMultipleSelection<String> evidenceCodes;
+
+    private CyApplicationManager applicationManager;
+    private EnrichmentCytoPanel enrichmentPanel;
+    private CyNetwork network;
+    private CyTable filteredEnrichmentTable;
+    private CyServiceRegistrar registrar;
+    private List<TermSource> categoryFilter = TermSource.getValues();
+
+    //public boolean removeOverlapping = false;
+
+    //public BoundedDouble overlapCutoff = new BoundedDouble(0.0, 0.5, 1.0, false, false);
```

Comment:
Nit: Let's remove the commented code if its not needed

Reply:
This code was already present so didn't meddle with it. Just changed the indentation.